solarforecastarbiter/io/reference_observations/tests/test_srml.py

```diff
@@ -29,8 +92,89 @@ def test_fetch(mocker, single_site, start, end, exp):
+        return_value=test_data)
+    data = srml.fetch('', single_site, start, end)
+    assert not [col for col in data.columns if '_flag' in col]
+    assert data['5161'].iloc[0] == 0.000235
+    assert data['5162'].iloc[0] == 0.000250
+
+
 def test_fetch_tz(single_site):
     start = dt.datetime(2019, 1, 1, tzinfo=dt.timezone.utc)
     with pytest.raises(TypeError):
         srml.fetch('', single_site, start,
                    dt.datetime(2019, 2, 1))
+
+
+@pytest.mark.parametrize('api_id,has_power', [
+    (94040.0, True),
+    (94291.0, False),
+])
+def test_adjust_parameters(api_id, has_power):
+    site = srml.adjust_site_parameters({
+        'extra_parameters': {
+            'network_api_id': str(api_id),
+            }
+    })
+    if has_power:
+        modeling_params = site['modeling_parameters']
+        assert modeling_params['ac_capacity'] == 0.02
+        assert modeling_params['dc_capacity'] == 0.02
+        assert modeling_params['ac_loss_factor'] == 0.0
+        assert modeling_params['dc_loss_factor'] == 0.0
+        assert modeling_params['surface_azimuth'] == 180.0
+        assert modeling_params['surface_tilt'] == 15.0
+        assert modeling_params['temperature_coefficient'] == 0.3
+        assert modeling_params['tracking_type'] == 'fixed'
+    else:
+        assert 'modeling_parameters' not in site
+
+
+def test_init_site_observations(
+        mocker, test_data, test_site):
+    mocker.patch(
+        'solarforecastarbiter.io.reference_observations.srml.request_data',
+        return_value=test_data)
+    mock_create_obs = mocker.patch(
+        'solarforecastarbiter.io.reference_observations.srml.common.'
+        'create_observation')
+    mock_chk_post = mocker.patch(
+        'solarforecastarbiter.io.reference_observations.srml.common.'
+        'check_and_post_observation')
+    mock_api = mocker.MagicMock()
+    srml.initialize_site_observations(mock_api, test_site)
+    assert mock_create_obs.call_count == 5
+    assert mock_chk_post.call_count == 2
+
+
+def request_data_test(mocker, exception, test_site, test_data):
+    get_func = mocker.patch(
+        'solarforecastarbiter.io.reference_observations.srml.iotools.'
+        'read_srml_month_from_solardat')
+    get_func.return_value = test_data
+    data = srml.request_data(test_site, 1, 1)
+    assert_frame_equal(data, test_data)
+
+
+@pytest.mark.parametrize('exception', [
+    pd.errors.EmptyDataError,
+    error.URLError,
+])
+def request_data_test_warnings(mocker, exception, test_site):
+    logger = mocker.patch(
+        'solarforecastarbiter.io.reference_observations.srml.iotools.'
+        'logger.warning')
+    get_func = mocker.patch(
+        'solarforecastarbiter.io.reference_observations.srml.iotools.'
+        'read_srml_month_from_solardat')
+    get_func.side_effect = exception()
+    data = srml.request_data(test_site, 1, 1)
+    assert logger.call_count == 5
+    assert data is None
```

Comment:
Are these tests actually run? They don't start with `test_` so I would assume not. Also codecov says no.